project1/backend/tests/ajvValidator.test.js

```diff
@@ -0,0 +1,110 @@
+import ajvValidator from '../utils/ajvValidator';
+import queueSchema from '../schemas/queueSchema';
+import resolutionSchema from '../schemas/resolutionSchema';
+
+const queueRequests = [
```

Comment:
In general, we shouldn't be testing messages.
We must test the types of errors what returned